idd/Energy+.idd.in

```diff
@@ -24111,6 +24145,70 @@ AirflowNetwork:Distribution:Component:ConstantPressureDrop,
       \minimum> 0.0
       \note Enter the pressure drop across this component.
 
+AirflowNetwork:Distribution:Component:OutdoorAirFlow,
+      \min-fields 3
+      \memo This object adopts outdoor air flow based on Controller:OutdoorAir object
```

Comment:
@mjwitte This object tells the program to use the OA flow rate from the Controller:OutdoorAir object when the mass flow rate > 0, and treat this component as a crack when no mass flow rate. Before this object, the model calculates the flow rate based on pressure resistance,